vision_system_node:
  ros__parameters:
    camera_topic: "/camera/image_raw"
    camera_info_topic: "/camera/camera_info"
    camera_frame: "camera"
  